.gitignore

```diff
@@ -2,6 +2,7 @@
 .DS_Store
 build
 .idea/**
+dist/**
```

Comment:
Minor: Should be enough to skip the wildcards and just have
```
.DS_Store
build
.idea
dist
```
Only requirement is that those names are unique, so we don't have any file called `.idea` or `dist`.